.packit.yaml

```diff
@@ -0,0 +1,59 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+packit_instances: ["stg"]
+upstream_project_url: https://github.com/fedora-copr/copr.git
+
+packages:
+
+  python:
```

Comment:
`python:` seems acceptable, but does the spelling matter for automation?  Having `python-copr` here would seem nicer (it's the actual package name, if we speak about `packages:`).

Reply:
Sure, I will rename it!